app/navigation/MDRNavigator.scala

```diff
@@ -64,12 +65,21 @@ class MDRNavigator @Inject() () extends Navigator {
     case WhatIsYourPostcodePage               => regime => _ => Some(routes.SelectAddressController.onPageLoad(CheckMode, regime))
     case SelectAddressPage                    => regime => addressWithoutID(CheckMode)(regime)
     case AddressUKPage                        => regime => addressWithoutID(CheckMode)(regime)
-
     case BusinessTypePage                     => regime => _ => Some(routes.UTRController.onPageLoad(CheckMode, regime))
     case UTRPage                              => isSoleProprietor(CheckMode)
-    case SoleNamePage                         => regime => _ => Some(routes.IsThisYourBusinessController.onPageLoad(CheckMode, regime))
-    case BusinessNamePage                     => regime => _ => Some(routes.IsThisYourBusinessController.onPageLoad(CheckMode, regime))
     case IsThisYourBusinessPage               => isThisYourBusiness(CheckMode)
+//    case RegistrationInfoPage                 => regime => ua => IndividualMatchRoutes(CheckMode)(regime)(ua)
+    case RegistrationInfoPage                 => regime => ua =>
```

Comment:
@soljeggers RegistrationInfoPage is used to save the registration info and use it in CYA, it should not be used for navigation